module-4/backend/week_one.md

```diff
@@ -1,21 +1,63 @@
 ## Week One - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
+  How to make a for loop; however, I wish there were more basic exercises to help us learn by repetition.
+
 2. What are some tools to help debug JavaScript code?
+
+  Debugger, pryjs and using console.log()
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+  Mocha and chai.
+
 4. What is the syntax for invoking a function?
+
+  ```javascript
+  function();
+  ```
+
 5. What's the difference between `==` and `===` in JavaScript?
-6. What's the difference between asynchronous and synchronous JavaScript? 
+
+  `==` is used for checking equality and `===` is checking for a true/false result
```

Comment:
type conversion is done with `==` and with `===`, there is not type conversion, it's just checking for equality. 

`'7' == 7` will return true
`'7' === 7` will return false